src/main/java/com/idrsolutions/microservice/BaseServlet.java

```diff
@@ -556,8 +576,12 @@ protected abstract void convert(final String uuid, final File inputFile, final F
      * @throws IOException on file not being writable
      */
     private File outputFile(final String filename, final String uuid, final byte[] fileBytes) throws IOException {
+
         final File inputDir = createInputDirectory(uuid);
-        final File inputFile = new File(inputDir, sanitizeFileName(filename));
+
+        final String cappedFileName = capFileName(filename, getServletContext());
```

Comment:
See suggestion in capFileName

```suggestion
        final Properties properties = (Properties) getServletContext().getAttribute(BaseServletContextListener.KEY_PROPERTIES);
        final String cappedFileName = capFileName(filename, properties);
```